src/annotator/test/frame-observer-test.js

```diff
@@ -170,13 +176,14 @@ describe('annotator/frame-observer', () => {
       fakeIFrame.setAttribute('src', 'http://my.dummy');
     });
 
-    it('waits for the iframe load event to be triggered if the document is blank', () => {
+    it('waits for the iframe to change the location and readyState if the initial document is blank', () => {
```

Comment:
When the navigation from `about:blank` to the URL specified in the `src` attribute happens, _the initial document is replaced_. In other words, `frame.contentDocument` (and `fakeIFrameDocument` here) changes to a different object. I think that needs to either be simulated correctly, or the test should be reworked to avoid relying on a fake.

Reply:
I correct it.